doc/qbk/4.0.BNF_parsing.qbk

```diff
@@ -0,0 +1,32 @@
+[/
+    Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+    Copyright (c) 2021 Alan de Freitas (alandefreitas@gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/CPPAlliance/url
+]
+
+[section:charset BNF Parsing]
+
+The file [@../../../../boost/url/bnf/parse.hpp `<boost/url/bnf/parse.hpp>`] contains the basic overloads of the function __bnf__parse__ for parsing BNF elements.
```

Comment:
No need to mention the file, and refs should use the full link

Reply:
> refs should use the full link
Why is that? Why are the macros such as 

```
[def __authority_view__         [link url.ref.boost__urls__authority_view `authority_view`]]
``` 

defined in 0.main.qbk then?

Reply:
> refs should use the full link
Why is that? Why are the macros such as 

```
[def __authority_view__         [link url.ref.boost__urls__authority_view `authority_view`]]
``` 

defined in 0.main.qbk then?